code/modules/hydroponics/plant_genes.dm

```diff
@@ -29,6 +29,23 @@
 	new_gene.mutability_flags = mutability_flags
 	return new_gene
 
+/*
+ * on_new_seed is called when seed genes are initialized on the /obj/seed.
+ *
+ * new_seed - the seed being created
+ */
+/datum/plant_gene/proc/on_new_seed(obj/item/seeds/new_seed)
+	return FALSE
+
+/*
+ * on_removed is called when the gene is removed from a seed.
+ * Also called when a seed is qdel'd (and all the genes are removed and deleted).
+ *
+ * old_seed - our seed, before being removed
+ */
+/datum/plant_gene/proc/on_removed(obj/item/seeds/old_seed)
+	return FALSE
```

Comment:
same as above, document what it means for it to return false. if it doesn't mean anything, it should return nothing instead